examples/ecommerce/src/App.mobile.css

```diff
@@ -170,6 +171,7 @@
   .ais-RefinementList-checkbox {
     height: 1.5rem;
     width: 1.5rem;
```

Comment:
`width` is not required with `min-width`.

Reply:
Right I was supposed to remove `width`. I'm doing the same to angular-IS. c1e37b2